driver-core/src/test/functional/com/mongodb/internal/async/client/ChangeStreamProseTest.java

```diff
@@ -143,7 +143,7 @@ public void testNoResumeErrors() {
 
         for (int errCode : asList(136, 237, 11601)) {
             try {
-                setFailPoint("getMore", errCode);
+                setFailPoint("getMore", errCode, false);
```

Comment:
I can't comment on the test definition because it wasn't changed, but this prose test was removed in the initial SPEC-1505 commit (https://github.com/mongodb/specifications/commit/f93d78191f3db2898a59013a7ed5650352ef6da8#diff-d4053e57c104db86756af1f322023a62L158). It will still pass because none of these are resumable errors, but it was removed because it doesn't make sense to specifically test these three errors, which are no different from any other non-resumable errors.

Reply:
I have removed the test.